functions/nodejs-apps.bash

```diff
@@ -60,6 +60,20 @@ nodejs_setup() {
 frontail_setup() {
```

Comment:
Make the theme a parameter to frontail_setup() in unattended mode
If called without a parameter then use the default theme